.github/stale.yml

```diff
@@ -0,0 +1,29 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+  - Epic
+
+  # Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
I think just because something is stale that shouldn't mean that we'll never fix it. `wontfix` sounds like "we've made the concious decision to not fix it", that's different from "we didn't have time to fix it". I'd rather use something like `stale` instead of `wontfix`

Reply:
fair enough